www/less/misc.less

```diff
@@ -53,6 +53,29 @@ a{
     padding-bottom:0px;
 }
 
+// Horizontal divider between title/description and lead section text
+#content_block_0_hr {
+    border:1px solid #999;
```

Comment:
This needs to be 1 by default for non-retina screens. Retina screens use 0.5 based on my other comment.